.ci/jenkins/Jenkinsfile.bump-up-version

```
@@ -0,0 +1,101 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.kie.jenkins.MavenCommand
+
+def repoName = 'kie-benchmarks'
+
+pipeline {
+    agent {
+        label 'kie-rhel8 && !built-in'
```

Thinking that it should be enough.
`kie-rhel8` is a 16G mem node, and we don't need it here IMO
```suggestion
        label 'rhel8 && !built-in'
```